internal/external-dns/plan/plan.go

```diff
@@ -635,6 +635,15 @@ func shouldUpdateProviderSpecific(desired, current *endpoint.Endpoint) bool {
 	return len(desiredProperties) > 0
 }
 
+func shouldUpdateGroup(desired, current *endpoint.Endpoint) bool {
+	currentGroup, hasCurrentGroup := current.Labels["group"]
+	desiredGroup, hasDesiredGroup := desired.Labels["group"]
+	if hasCurrentGroup && hasDesiredGroup {
+		return currentGroup != desiredGroup
+	}
+	return hasCurrentGroup != hasDesiredGroup
+}
+
```

Comment:
Hmm OK, i guess we need this so that changes to the group result in an update. I have kind of looked into this before, wondering if it would be better to make this more generic and update on any label changes instead of hard coding specific labels like this.

Nothing to do here, this will work, just commenting.

Reply:
Undo these changes in favour of https://github.com/Kuadrant/dns-operator/pull/643 once merged.